src/docs/topics/docker/README.md

```diff
@@ -0,0 +1,194 @@
+            - postgresql
+    sqlserver:
+        image: "mcr.microsoft.com/mssql/server"
+        environment:
+            SA_PASSWORD: "P@ssw0rd!123456"
+            ACCEPT_EULA: "Y"
+    mysql:
+        image: mysql:latest
+        restart: always
+        environment:
+            MYSQL_DATABASE: 'orchardcore_database'
+            MYSQL_USER: 'orchardcore_user'
+            MYSQL_PASSWORD: 'orchardcore_password'
+            MYSQL_ROOT_PASSWORD: 'root_password'
+        ports:
+            - '3306:3306'
+        expose:
+            - '3306'
+        volumes:
+            - mysql-data:/var/lib/mysql
+    postgresql:
+        image: postgres:latest
+        volumes:
+            - postgresql-data:/var/lib/postgresql/data
+        ports:
+            - 5432:5432
+        environment:
+            POSTGRES_USER: orchardcore_user
+            POSTGRES_PASSWORD: orchardcore_password
+            POSTGRES_DB: orchardcore_database
+volumes:
+    mysql-data:
+    postgresql-data:
+
+```
+## Prune intermediate images
+
+### When using `docker-compose` command : 
+
+```cmd
+REM Builds images if they are not already built
+docker-compose build
+
+REM Prune intermediate remaining images
+docker image prune -f --filter label=stage=build-env
+
+REM Start all containers
+docker-compose up
+```
+
+We added some commands examples to prune intermediate images because our `Dockerfile` uses an intermediate image to do a `dotnet publish` of our source code. If you don't prune your intermediate images ; over time it can certainly take some significant amount of disk space. We label those intermediate images with `stage=build-env` so that they can be removed easily.
+
+## Create tenants automatically (Autosetup feature)
+
+TODO
+
+See :
+
+[Database shell configurations](../../reference/core/Shells/README.md#database-shells-configuration-provider)
+
+[Autosetup feature PR](https://github.com/OrchardCMS/OrchardCore/pull/4567)
+
+## How can I run my Orchard Core Docker containers over HTTPS?
+
+[Hosting ASP.NET Core Images with Docker over HTTPS](https://github.com/dotnet/dotnet-docker/blob/master/samples/host-aspnetcore-https.md)
+
+## How can I target my Docker images against a specific environment?
+
+Here you can find a list of different environments with a Dockerfile example for each of them.
+
+[.NET Core SDK](https://hub.docker.com/_/microsoft-dotnet-sdk/)  
+[ASP.NET Core Runtime](https://hub.docker.com/_/microsoft-dotnet-aspnet/)  
+
+## Why does Orchard Core source code uses a different Dockerfile?
+
+### Dockerfile-CI, Dockerfile-CI.gitignore
+
+For building and testing Orchard Core we use Github Actions (CI). Github Actions allows us to create Docker images and containers but the execution time on them are slower than on the actual CI. So for that matter we don't use an intermediate image for building Orchard Core on the CI. Though it perfectly makes sense to do this locally for yourself as performance should not be limited by any means.
```

Comment:
`For building and testing Orchard Core ...` => rephrase.
Something like: `Github Actions is used as the Continuous Integration tool to build and test the branches.`

`but the execution time on them are slower than ...` => fix singular/plural and rephrase

`So for that matter we don't ...` => Use commas to clearly separate the different parts in the sentences.

`by any means` => `at all`, may be.

As a French speaker, struggling with English, I know we have sometimes the bias to translate word by word and then to rephrase a sentence to make it sound better.

I hope my remarks are helping. 
Thank you for your work.

Reply:
Rephrased

Reply:
I think it comes from the fact that they reverse everything in sentences. And if we think in french then we often making spelling mistakes.

Example: 

Our source code repository includes a `Dockerfile`
Notre référentiel de code source inclus un `Dockerfile`